eng/scripts/InstallVisualStudio.ps1

```diff
@@ -32,8 +36,10 @@
 param(
     [ValidateSet('BuildTools','Community', 'Professional', 'Enterprise')]
     [string]$Edition = 'Enterprise',
-    [ValidateSet('Release', 'Preview')]
+    [ValidateSet('Release', 'Preview', 'Preview')]
```

Comment:
Needs less cowbell despite the beauty of https://www.youtube.com/watch?v=cVsQLlk-T0s
```suggestion
    [ValidateSet('Release', 'Preview')]
```